src/overlay/ItemFetcher.h

```diff
@@ -14,19 +14,6 @@
 #include <util/optional.h>
 #include "util/HashOfHash.h"
 
-/*
```

Comment:
This comment was mostly made invalid in this commit: https://github.com/stellar/stellar-core/commit/7a86fff5ed8ad009faed7856c7e9b5e411c305ef:
* ItemFetcher does not return instances of the Tracker class
* there are no longed shared_ptrs of Tracker instances

Remaining valid sentences were moved directly before ItemFetcher class:

+/**
 + * @class ItemFetcher
 + *
 + * Manages asking for Transaction or Quorum sets from Peers
 + *
 + * The ItemFetcher keeps instances of the Tracker class. There exists exactly
 + * one Tracker per item. The tracker is used to maintain the state of the
 + * search.
 + */